### DISCUSSION TITLE
Archive on delete

### DISCUSSION THREAD

**Discussion by manuelwehrli** [directus/directus#4814] [Feature Requests]
Could we have an option to map the delete endpoint/mutation to archive (soft-delete) instead of deleting permanently? The use case would be to offer the standard api to delete an item (no need for the consumer to know about the archive field) but keep it in the backend for easy on-demand recovering or store it for historic or legal reason. For now, the only way seems to be to disable delete permission, disable read permission of the archive field, filter archived items out and permit to update the archive field from false to true. This way, we kind of expose to api consumers that we do soft-delete in the backend, what we might not want them to know.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
